src/main/java/com/mojang/brigadier/builder/ArgumentBuilder.java

```diff
@@ -39,40 +70,116 @@ public T then(final CommandNode<S> argument) {
         return getThis();
     }
 
+    /**
+     * Returns all registered child commands, which are the registered arguments.
+     *
+     * @return all registered child commands, which are the registered arguments.
+     */
     public Collection<CommandNode<S>> getArguments() {
         return arguments.getChildren();
     }
 
+    /**
+     * Sets the command that will be executed by the built {@link CommandNode}.
+     *
+     * @param command the {@link Command} to execute
+     * @return this object
+     */
     public T executes(final Command<S> command) {
         this.command = command;
         return getThis();
     }
 
+    /**
+     * Returns the {@link Command} the built {@link CommandNode} will execute.
```

Comment:
I added it to the doc of the builder and the command node.

There does not seem to be any standard pulled in however ( like findbugs, intelliJ, google, JSR, ...) or my IDE doesn't suggest it.
I don't really want to make that decision, but it would be nice to have, yea.

Reply:
Yes you are right, it is better to wait for the devs to change that then.